src/components/BccButton/BccButton.vue

```diff
@@ -106,24 +106,24 @@ const buttonClassVariants = cva(
         color: "default",
         disabled: false,
         class: [
-          "bg-button-primary-default text-button-primary-default hover:bg-button-primary-hover active:bg-button-primary-pressed active:text-button-primary-default focus:outline-none focus:ring focus:bg-button-primary-focused focus:ring-silver-tree-700 focus:ring-offset-2",
+          "bg-button-primary-default text-button-primary-default hover:bg-button-primary-hover active:bg-button-primary-pressed active:text-button-primary-default focus:border-none focus:ring focus:bg-button-primary-focused focus:ring-silver-tree-700 focus:ring-offset-2",
```

Comment:
This should remain `outline-none` as it's there to block the native browser outline appearing on focus, as we use a custom `ring` for that.

Right now this makes the primary and tertiary buttons jump around on focus as the transparent border is removed.

![Screen Recording 2023-03-22 at 11 44 25](https://user-images.githubusercontent.com/39056299/226880043-324f5bc2-995a-4194-81ed-7649713009f3.gif)